SiraUtil/Tools/FPFC/FPFCToggle.cs

```diff
@@ -181,11 +181,8 @@ private void DisableFPFC()
             foreach (var listener in _fpfcListeners)
                 listener.Disabled();
 
-            _siraLog.Debug(IsProcessRunning("vrserver") ? "VR Server is running" : "VR Server is not running");
-            _siraLog.Debug(IsProcessRunning("vrcompositor") ? "VR Compositor is running" : "VR Compositor is not running");
-            _siraLog.Debug(Environment.GetCommandLineArgs().Any(s => s.ToLower() == "oculus") ? "Running as vrmode Oculus" : "Running as vrmode Steam");
-            bool startXR = (IsProcessRunning("vrserver") && IsProcessRunning("vrcompositor")) || Environment.GetCommandLineArgs().Any(s => s.ToLower() == "oculus"); // If we're running Oculus, we need to start XR
-            if (startXR) InitializeXRLoader();
+            if (IsProcessRunning("vrserver") && IsProcessRunning("vrcompositor")) // Not sure how to check for Oculus if vrmode oculus is not working anymore
```

Comment:
Realisticaly there is no need to reinitialize openxr, since the game can function fine in FPFC without it, but this makes it impossible for oculus only users to start the game in FPFC and still have the headset bind on toggle...

One way would be to see if there is an oculus specific process that could be checked, but I dont own an oculus headset anymore so I cant see if there is one